.buildkite/job-build-test-all-debug.yml

```
@@ -0,0 +1,36 @@
+## 🏠/.buildkite/pipeline.yml
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0 and the following additional limitation. Functionality enabled by the
+# files subject to the Elastic License 2.0 may only be used in production when
+# invoked by an Elasticsearch process with a license key installed that permits
+# use of machine learning features. You may not use this file except in
+# compliance with the Elastic License 2.0 and the foregoing additional
+# limitation.
+env:
+  BUILD_OS: "linux"
+  BUILD_ARCH: "x86_64"
+  BUILD_TYPE: "Debug"
+  BUILD_SNAPSHOT: "yes"
+  CMAKE_FLAGS: "-DCMAKE_TOOLCHAIN_FILE=cmake/linux-x86_64.cmake"
+  ML_DEBUG: 1
+steps:
+  - label: "Testing of the Machine Learning main branch with full debug and assertions."
+    timeout_in_minutes: 180
+    agents:
+      cpu: "6"
+      ephemeralStorage: "20G"
+      memory: "64G"
+        image: "docker.elastic.co/ml-dev/ml-linux-build:24.bk"
```

```suggestion
        image: "docker.elastic.co/ml-dev/ml-linux-build:25"
```

Or if that doesn't work then whatever stops it working needs to be fixed.